tlslite/constants.py

```diff
@@ -124,7 +124,8 @@ class ContentType(TLSEnum):
     alert = 21
     handshake = 22
     application_data = 23
-    all = (20, 21, 22, 23)
+    HEARTBEAT = 24  # RFC 6520
```

Comment:
all the other identifiers are lower case because their names come from RFCs, so for consistency this one should also be lowercase